lib/matplotlib/dviread.py

```diff
@@ -58,10 +58,70 @@
 
 # The marks on a page consist of text and boxes. A page also has dimensions.
 Page = namedtuple('Page', 'text boxes height width descent')
-Text = namedtuple('Text', 'x y font glyph width')
 Box = namedtuple('Box', 'x y height width')
 
 
+# Also a namedtuple, for backcompat.
+class Text(namedtuple('Text', 'x y font glyph width')):
+    """
+    A glyph in the dvi file.
+
+    The *x* and *y* attributes directly position the glyph.  The *font*,
+    *glyph*, and *width* attributes are kept public for back-compatibility, but
+    users wanting to draw the glyph themselves are encouraged to instead load
+    the font specified by `font_path` at `font_size`, warp it with the effects
+    specified by `font_effects`, and load the glyph specified by `glyph_spec`.
+    """
+
+    def _get_pdftexmap_entry(self):
+        return PsfontsMap(find_tex_file("pdftex.map"))[self.font.texname]
```

Comment:
Performance: Does it make sense to cache this result?

Reply:
There's already a cache in PsfontsMap instantiation, so no need to add another layer here.